docs/src/app/components/master.jsx

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
+import Title from 'react-title-component';
+
 import AppBar from 'material-ui/lib/app-bar';
 import IconButton from 'material-ui/lib/icon-button';
 import {Spacing} from 'material-ui/lib/styles';
-import {
-  StylePropable,
-  StyleResizable,
-} from 'material-ui/lib/mixins';
+import {StylePropable, StyleResizable} from 'material-ui/lib/mixins';
```

Comment:
If I had a clue what it is or how to change it maybe...

On second thoughts, having watched @newoga's marathon - maybe not! :laughing: